flags.go

```diff
@@ -43,22 +53,32 @@ var flagsMtx = sync.RWMutex{}
 // whether or not the flag should be considered
 // enabled. It returns false if no flag with the specified
 // name is found
-func Enabled(name string) bool {
+func Enabled(name string) (enabled bool) {
+	defer func() {
+		var gauge float64
+		if enabled {
+			gauge = 1
+		}
+		stats.Gauge("goforit.flags.enabled", gauge, []string{fmt.Sprintf("flag:%s", name)}, .1)
```

Comment:
I share the cardinality FUD: We may have short-lived flags, but there may still be a lot of them at any one time across all sorts of hosts in our fleet.